src/js/helpers.js

```diff
@@ -168,7 +168,7 @@ function assignments (node) {
     // Find assignments table, get it's rows, take out the header and legend rows.
     [...node.querySelector('table[align=center').querySelectorAll('tr')].slice(1, -1).forEach((e, i) => {
         const curr = e.querySelectorAll('td');
-        const assignment = new Assignment(curr[2].innerText, curr[curr.length - 1].innerText, i);
+        const assignment = new Assignment(curr[2] ? curr[2].innerText : "", curr[curr.length - 1] ? curr[curr.length - 1].innerText : "", i);
```

Comment:
Just looks cleaner but your choice :)
```suggestion
        const assignment = new Assignment(curr[2]?.innerText || "", curr[curr.length - 1]?.innerText || "", i);
```